apis/identity/v1alpha1/iamgrouppolicyattachment_types.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+)
+
+// IAMGroupPolicyAttachmentParameters define the desired state of an AWS IAMGroupPolicyAttachment.
+type IAMGroupPolicyAttachmentParameters struct {
+
+	// PolicyARN is the Amazon Resource Name (ARN) of the IAM policy you want to
+	// attach.
+	// +immutable
+	PolicyARN string `json:"policyArn,omitempty"`
+
+	// PolicyARNRef references an IAMPolicy to retrieve its Policy ARN.
+	// +optional
+	PolicyARNRef *runtimev1alpha1.Reference `json:"policyArnRef,omitempty"`
+
+	// PolicyARNSelector selects a reference to an IAMPolicy to retrieve its
+	// Policy ARN
+	// +optional
+	PolicyARNSelector *runtimev1alpha1.Selector `json:"policyArnSelector,omitempty"`
+
+	// GroupName presents the name of the IAMGroup.
+	// +immutable
+	GroupName string `json:"groupName,omitempty"`
```

Comment:
same here, the filed should be annotated with `+optional` and be of pointer type.